packages/outputarea/src/widget.ts

```diff
@@ -859,10 +893,19 @@ export class Stdin extends Widget implements IStdin {
   /**
    * The value of the widget.
    */
-  get value() {
+  get value(): Promise<string> {
     return this._promise.promise.then(() => this._value);
   }
 
+  /**
+   * Submit and get the value of the Stdin widget.
+   */
+  getValue(): Promise<string> {
+    console.log('WHAT THE FUCK!');
```

Comment:
Quick note - this was recently reported to us by a user, and @choldgraf already opened a [repo2docker issue](https://github.com/jupyterhub/repo2docker/issues/1080) to track it and quickly propagate an update to 3.1.11. Unfortunately we happened to update the Binder images to the one that has this code live in it and it means that for a bit, _all_ binder users will see this.

From the logs, it seems this line was introduced on June 13 in [29e50](https://github.com/jupyterlab/jupyterlab/commit/29e50c6225260b0c57c0ab9af68cd873fa8898df#diff-4d0f96dfb060e341e538a737407cb92ad9a1003d88037f844a289e11134837abR904), and it was removed on Augusst 6 in in [ae1846](https://github.com/jupyterlab/jupyterlab/commit/ae1846768e7cb22fbef63cce325fe5ad395b655f#diff-4d0f96dfb060e341e538a737407cb92ad9a1003d88037f844a289e11134837abL919) (thanks @echarles!).

@goanpeca for future reference, it is **not OK**  to put code like this into a submitted PR. I presume this was unintentional and it was left in by accident from internal debugging, but I hope the lesson is that it's never a good idea to leave code like that even in a personal commit, as it's very easy to forget and have it slip by unnoticed for a while. 

When you need log messages that stand out, using things like `*** DBG  - ..... ***` potentially with longer/more visible markers if desired, is both easier to see and doesn't create these issues.

Nothing else to do on this PR since the code is already removed, but I still wanted to leave this note here so that folks keep this in mind and avoid similar incidents in the future. Thanks!

Reply:
Hi @fperez, yes it was an honest mistake, trying to debug an issue.

I am aware this is not appropriate language, my apologies to the community, this was really an oversight on my side when I made the changes to the original PR.

Note taken